docs/components/guide-drawer.md

```diff
@@ -3,29 +3,28 @@
 The Guide Drawer helps a user learn about the product by letting them use the real UI.
 
 ## Visual Examples
-| Example | Description|
-| :---: |---|
-| ![](https://drive.google.com/uc?id=19uRGfYFWxSn7sRrlVIAXy5MsV7fcucQ1) | Add options to the drawer so that the user can choose where to go next. |
-| ![](https://drive.google.com/uc?id=1tUvdz24bEEJalNCHKez3SyIW6dO5H-kL) | A typical guide step includes instructive copy telling the user how to use the normal product interface and can include external links to more information. |
-| ![](https://drive.google.com/uc?id=1lS44XB8knV1ORL7dkcas8_02VA02M-eJ) | Drawer content can include things like images, illustrations or videos. |
+| Default | Adding Options | Adding Media |
+| --- |---| --- |
+| ![](https://drive.google.com/uc?id=19uRGfYFWxSn7sRrlVIAXy5MsV7fcucQ1) | ![](https://drive.google.com/uc?id=1tUvdz24bEEJalNCHKez3SyIW6dO5H-kL) | ![](https://drive.google.com/uc?id=1lS44XB8knV1ORL7dkcas8_02VA02M-eJ) |
+| Add options to the drawer so that the user can choose where to go next. | A typical guide step includes instructive copy telling the user how to use the normal product interface and can include external links to more information. | Drawer content can include things like images, illustrations or videos. |
 
 ## Best Practices
 
-#### Guide Drawers should...:
+### Guide Drawers should...:
 
 - ✅ Stay fixed to the right of the viewport on desktop, 12px of spacing below the header and on the right and bottom sides
```

Comment:
I'm guessing you're calling out the 12px offset here because it's not necessarily demonstrated in the mockup and possibly also not an intrinsic CSS style in the component? If so, I think that level of detail makes sense.

I'm thinking mostly about maintenance and in what situations we should includew this level of detail (versus relying on the source file).

Reply:
That does seem like it should be built-in, good point. I think we can remove this one then.